src/index.html

```diff
@@ -3,11 +3,40 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="img"
+      >
+      <p class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
```

Comment:
```suggestion
      <h2 class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
It's better to use semantic tags for headings